### PR TITLE
portal/brand: Added invoicing forms

### PR DESCRIPTION
**Pull request by fmendozzac**
#### Type Of Change 
- [ ] Small bug fix
- [x] New feature or enhancement
- [ ] Breaking change

#### Checklist:
- [x] Commits are named and have tag following [commit rules](https://github.com/irontec/ivozprovider/blob/bleeding/doc/dev/en/commits.md)
- [x] Commits are split per component (schema, web/admin, kamusers, agis, ..)
- [x] Changes have been tested locally
- [ ] Fixes an existing issue (Fixes #XXXX) 
- [ ] Upport from existing Pull request #XXXX

### CODE REVIEW

**Comment by ironArt3mis** [comment]
Rebase completed.

File: web/portal/brand/src/entities/Invoice/Form.tsx [hunk 1]
```diff
@@ -18,23 +17,20 @@ const Form = (props: EntityFormProps): JSX.Element => {
...
+      legend: '',
+      fields: ['number', 'company', 'invoiceTemplate'],
+    },
+    {
+      legend: '',
+      fields: ['inDate', 'outDate'],
+    },
+    {
+      legend: '',
+      fields: ['total', 'taxRate', 'totalWithTax'],
```
Comment (mmadariaga) [inline web/portal/brand/src/entities/Invoice/Form.tsx:29]: taxRate must be editable

File: web/portal/brand/src/entities/Invoice/Form.tsx [hunk 2]
```diff
@@ -18,23 +17,20 @@ const Form = (props: EntityFormProps): JSX.Element => {
...
+    },
+    {
+      legend: '',
+      fields: ['status', 'pdf'],
```
Comment (mmadariaga) [inline web/portal/brand/src/entities/Invoice/Form.tsx:33]: pdf must be read only

**Review by mmadariaga — CHANGES_REQUESTED** [review]

File: web/portal/brand/src/entities/FixedCost/FixedCost.tsx [hunk 3]
```diff
@@ -19,6 +19,7 @@ const properties: FixedCostProperties = {
   },
   cost: {
     label: _('Cost'),
+    type: 'integer',
```
Comment (mmadariaga) [inline web/portal/brand/src/entities/FixedCost/FixedCost.tsx:22]: This should not be necessary in ivoz-ui 0.10.3

**Review by mmadariaga — CHANGES_REQUESTED** [review]

**Review by mmadariaga — APPROVED** [review]

**Pull request merged**